src/main/java/com/senseidb/clue/util/MatchSomeDocsQuery.java

```diff
@@ -12,7 +12,17 @@
 public abstract class MatchSomeDocsQuery extends Query {
   
   protected abstract boolean match(int docId);
-  
+
```

Comment:
it won't compile without these, 'equals' and 'hashCode' are not abstract method in 6.2, so have to override them.